### PR TITLE
MVP & Stretch Complete

### PR DESCRIPTION
**Pull request by LadyKerr**

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -12,9 +12,13 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+    return {
+      count: initialState.count++
```
Comment (sebaghdev) [inline src/reducers/index.js:16]: You need to spread state or you will have error where when you try to increment, it won't do anything until you press the button a second time. It also will stay at 0 when you press the async button unless you press twice.

```
      return{
        ...state,
        state: state.count++
      };
```

File: src/reducers/index.js [hunk 2]
```diff
@@ -12,9 +12,13 @@ const initialState = {
...
+    }
     case DECREMENT:
-    // Fill in the body of this case
+    return {
+      count: initialState.count--
```
Comment (sebaghdev) [inline src/reducers/index.js:20]:
```
    return{
      ...state,
      state: state.count--
    }
```

File: src/components/Counter.js [hunk 3]
```diff
@@ -4,37 +4,36 @@ import { increment, decrement } from '../actions';
 
 class Counter extends Component {
     incrementIfOdd = () => {
-        // Stretch Problem: Implement an increment function that
-        // only increments if the counter value is odd
+        if(this.props.count % 2) {
+            return {count: this.props.increment()}
```
Comment (sebaghdev) [inline src/components/Counter.js:8]: No need for return.

```
            this.props.increment()
```

File: src/reducers/index.js (hunk 1 above)
Comment (LadyKerr) [reply src/reducers/index.js:16]: oh I had this. Guess I deleted before I committed, Will update!

File: src/components/Counter.js (hunk 3 above)
Comment (LadyKerr) [reply src/components/Counter.js:8]: Thank you!

**Pull request merged**